.changeset/ninety-boats-yawn.md

```diff
@@ -0,0 +1,8 @@
+---
+"@justeattakeaway/pie-webc-core": minor
+"@justeattakeaway/pie-components-config": minor
```

Comment:
Do we need a patch bump for the components so that this is picked up?

Reply:
I'm unsure, this doesn't give consumers anything so I think not?